js/google_ima.js

```diff
@@ -937,7 +937,10 @@ require("../html5-common/js/utils/utils.js");
 
           //iphone performance is terrible if we don't use the custom playback (i.e. filling in the second param for adDisplayContainer)
           //also doesn't not seem to work nicely with podded ads if you don't use it.
-          _IMAAdDisplayContainer = new google.ima.AdDisplayContainer(_amc.ui.adWrapper[0],
+
+          //for IMA, we always want to use the plugins element to house the IMA UI. This allows it to behave
+          //properly with the Alice skin.
+          _IMAAdDisplayContainer = new google.ima.AdDisplayContainer(_amc.ui.pluginsElement[0],
```

Comment:
Should all ad managers do this or do others use adWrapper?  Should pluginsElement be the new adWrapper?